katello/katello-backup

```diff
@@ -299,19 +333,27 @@ def offline_backup(database, dir_path = nil)
       puts "Done."
     end
   when 'pgsql'
+    foreman_config = foreman_db_config
     dir_path ||= '/var/lib/pgsql/data'
-    FileUtils.cd dir_path do
-      puts "Backing up postgres db..."
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pgsql_data.tar')} --listed-incremental=#{File.join(@dir, '.postgres.snar')} --transform 's,^,var/lib/pgsql/data/,S' -S *")
-      puts "Done."
+    if is_remote_db?(foreman_config['host'])
+      puts "Backup of #{dir_path} is not supported for remote databases. Doing online backup instead... "
```

Comment:
s/Doing online backup instead/Doing postgres online backup instead/g ? Just to be clear